backend/danswer/background/celery/celery_app.py

```diff
@@ -418,11 +533,12 @@ def kombu_message_cleanup_task_helper(ctx: dict, db_session: Session) -> bool:
             if result.rowcount > 0:  # type: ignore
                 ctx["deleted"] += 1
         else:
-            task_name = payload["headers"]["task"]
-            logger.warning(
-                f"Message found for task older than {ctx['cleanup_age']} days. "
-                f"id={task_id} name={task_name}"
-            )
+            # task_name = payload["headers"]["task"]
```

Comment:
the warning no longer is interesting since celery isn't going to clear out old tasks in postgres. But I think we can just remove the logging altogether.